src/services/resolvers/bonfida-resolver.service.ts

```diff
@@ -21,6 +21,9 @@ export class BonfidaResolverService extends ResolverService {
     }
 
     async resolve(domain: string): Promise<Account[]> {
+        if (!domain.endsWith(".sol")) {
+            throw Error(`Bonfida Error: ${domain} is not supported`);
```

Comment:
Put inside try catch?)
And you won't need to add "Bonfida Error", just `"${domain} is not supported"`